launcher/ui/MainWindow.cpp

```diff
@@ -1491,18 +1505,8 @@ void MainWindow::on_actionKillInstance_triggered()
     }
 }
 
-void MainWindow::on_actionCreateInstanceShortcut_triggered()
-{
-    if (!m_selectedInstance)
-        return;
-    auto desktopPath = FS::getDesktopDir();
-    if (desktopPath.isEmpty()) {
```

Comment:
Can we have this protection back(there are some systems where the applications or desktop path is empty

Reply:
If you do not want it to be here you can make the action hidden/enabled if it has the paths or not